src/tweens/typedefs/TimelineBuilderConfig.js

```diff
@@ -9,7 +9,7 @@
  * @property {number} [delay=0] - The number of milliseconds to delay before the tween will start. Each individual Tween configuration can override this value.
  * @property {array} [easeParams] - Optional easing parameters. Each individual Tween configuration can override this value.
  * @property {(string|function)} [ease='Power0'] - The easing equation to use for each tween. Each individual Tween configuration can override this value.
- * @property {number} [hold=0] - The number of milliseconds to hold each tween before yoyoing. Each individual Tween configuration can override this value.
+ * @property {number} [hold=0] - The number of milliseconds to hold each tween before yoyo'ing. Each individual Tween configuration can override this value.
```

Comment:
As with falsey, this form is dominant in this repository. While there's an argument against using `'` in this manner, I don't generally try to enforce global rules (and I do try to identify cases where my changes may be controversial).

I'm happy to drop any particular commits as directed....

Reply:
maybe it will be better to write "before `yoyo`." ?

Reply:
I won't make a change unless someone confirms they want it. I don't understand this context enough to have a real opinion. While I vaguely understand what a `tween` is and can imagine a concept of `yoyo` (technically the original word is `yo-yo` fwiw) -- I'm not sure how this differs from `spring` in Apple/Fluid design, and I really don't understand how this project wants to / believes the word / word family can be used. (I'm also not sure who owns this project.)

Reply:
I'm happy with the apostrophe.